### PR TITLE
Fixed developer utility UI and made some tweaks to the chapter Arabic names and Navigation drawer.

### PR DESCRIPTION
**Pull request by microart-inc**
### Summary
Reworked developer utility UI to better match the rest of the app. Also vertically centered the Arabic surah names on the right-hand side on the main page. Some links in the navigation sidebar were also not highlighting on hover, so those were fixed as well.

### Test Plan
The developer utility looks much cleaner and still retains the same functionality. The navigation drawer links highlight the primary blue color on hover, and the Arabic surah names are centered vertically.

### Screenshots

| Before | After |
| ------ | ------ |
| https://imgur.com/a/PIKPVOo | https://imgur.com/a/kXJHDqn |

### CODE REVIEW

File: src/components/DeveloperUtility/DeveloperUtility.module.scss [hunk 1]
```diff
@@ -1,25 +1,49 @@
...
-  top: calc(var(--spacing-mega) + var(--spacing-medium));
-  right: calc(var(--spacing-mega) + var(--spacing-medium));
-  height: calc(2*var(--spacing-mega));
-  width: calc(2*var(--spacing-mega));
-  
-  z-index: var(--z-index-max);
-  border-radius: 100%;
-  box-shadow: var(--shadow-regular);
-  text-align: center;
+	background: rgb(230, 230, 230);
```
Comment (tifa2UP) [inline src/components/DeveloperUtility/DeveloperUtility.module.scss:2]: Thank you microArt for your PR! Love the new functionality.

We avoid using hardcoded values for colors, sizing etc. Please check the theme.scss file and see which variable most closely aligns with your desired UI. 

It applies for all the custom values in this PR.

**Review by tifa2UP — CHANGES_REQUESTED** [review]

**Review by microart-inc — COMMENTED** [review]
Moved all sizing and colors from hardcoded values to variables provided in theme.scss.

**Comment by tifa2UP** [comment]
Thank you, this looks great. Can you pull master and resolve the merge conflicts? Good to go afterwards insha'allah

**Comment by microart-inc** [comment]
Fixed merge conflicts with 'master'

**Comment by tifa2UP** [comment]
Can you pull the latest master once more? I see a few conflicting files. Also feel free to ping me on our discord: https://discord.gg/PJ435cSk

src/components/DeveloperUtility/DeveloperUtility.module.scss
src/components/Navbar/NavigationDrawer/NavigationDrawerItem.module.scss
src/components/chapters/ChapterBlock.module.scss
src/styles/theme.scss

**Comment by tifa2UP** [comment]
Merged! Thank you microart

**Pull request merged**